### PR TITLE
Feature videoconverter

### PR DESCRIPTION
**Pull request by JessickaMoya**

### CODE REVIEW

File: src/main/java/org/fundacion/jala/converter/service/videoclasses/Converter.java [hunk 1]
```diff
@@ -36,7 +36,7 @@ public void convertVideo(final String pathFile) {
         pathOutput = adaptPath.substring(0, (adaptPath.lastIndexOf("archive"))) + "archive\\";
         String fCommand = startFirstCommand + adaptPath + " ";
         String parameters = changeResolution() + changeFrameRate() + removeAudio();
-        String theCommand = fCommand + parameters + pathOutput + output  + "\"" + generateATumbnail() + " -y";
+        String theCommand = fCommand + parameters + pathOutput + output  + "\"" + " -y";
         System.out.println(theCommand);
         try {
```
Comment (danielaSantaCruz) [inline src/main/java/org/fundacion/jala/converter/service/videoclasses/Converter.java:44]: Logs are missing on the try - catch in this class

File: src/main/java/org/fundacion/jala/converter/service/videoclasses/Converter.java [hunk 2]
```diff
@@ -73,34 +80,20 @@ private String changeResolution() {
     }
 
     /**
-     * Generates a json file with the output video metadata
+     * Generates a input video tumbnail
```
Comment (Raymundogs) [inline src/main/java/org/fundacion/jala/converter/service/videoclasses/Converter.java]: it should be thumbnail

File: src/main/java/org/fundacion/jala/converter/service/videoclasses/Converter.java (hunk 1 above)
Comment (JessickaMoya) [reply src/main/java/org/fundacion/jala/converter/service/videoclasses/Converter.java:44]: Thanks. I just did it

File: src/main/java/org/fundacion/jala/converter/service/videoclasses/Converter.java (hunk 2 above)
Comment (JessickaMoya) [reply src/main/java/org/fundacion/jala/converter/service/videoclasses/Converter.java]: Thanks. I just fix it

**Review by danielaSantaCruz — APPROVED** [review]

**Review by Raymundogs — APPROVED** [review]

File: src/main/java/org/fundacion/jala/converter/service/videoclasses/Converter.java [hunk 3]
```diff
@@ -36,20 +39,24 @@ public void convertVideo(final String pathFile) {
         pathOutput = adaptPath.substring(0, (adaptPath.lastIndexOf("archive"))) + "archive\\";
         String fCommand = startFirstCommand + adaptPath + " ";
```
Comment (edsonarios) [inline src/main/java/org/fundacion/jala/converter/service/videoclasses/Converter.java]: "fCommand" name is not descriptive

**Review by edsonarios — APPROVED** [review]
Good job

**Pull request merged**